sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -420,7 +421,7 @@ export interface ServiceBusReceivedMessageWithLock extends ServiceBusReceivedMes
     defer(propertiesToModify?: {
         [key: string]: any;
     }): Promise<void>;
-    renewLock(): Promise<Date>;
+    renewMessageLock(): Promise<Date>;
```

Comment:
Though I had made the request to make this change from `renewLock()` to `renewMessageLock()`, I am not second guessing it...

In .Net and Java, this renaming is helpful to differentiate between renewing lock on message vs session, because both methods are on the receiver. In JS & Python, we have the method to renew message lock on the message itself. So, having the word "message" in the method name can be considered redundant.

cc @HarshaNalluru, @richardpark-msft

Reply:
Makes sense, having "renewMessageLock" would also call for making `complete` -> `completeMessage`, etc for consistency.
I agree that this is redundant. 

Let's not have "message" word in the method name.

Reply:
Agreed.

@mohsin-mehmood, Please revert the change to this method

Reply:
Done